### PR TITLE
Re-enable CI environments

### PR DESCRIPTION
**Pull request by ChrisRackauckas**

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -6,6 +6,40 @@ on:
...
+      - uses: julia-actions/julia-buildpkg@v1
+      - uses: julia-actions/julia-runtest@v1
+        env:
+          GROUP: ${{ matrix.group }}
+      - uses: julia-actions/julia-processcoverage@v1
+      - uses: codecov/codecov-action@v1
+        with:
+          file: lcov.info
+
+
```
Comment (devmotion) [inline .github/workflows/CI.yml:42]: There already exists a job of name `test`. It should be removed or extended.

Comment (ChrisRackauckas) [reply .github/workflows/CI.yml:42]: I don't know what you mean.

Comment (devmotion) [reply .github/workflows/CI.yml:42]: The workflow contains two jobs of the same name, one for linux and one for Windows. They should be merged.

Comment (devmotion) [reply .github/workflows/CI.yml:42]: That's also the reason for why the workflow is no valid and not even the tests on Github run.

**Comment by ChrisRackauckas** [comment]
The other repos get the name in the links: is there a way to do that here?

@christopher-dG

**Comment by devmotion** [comment]
There exists already a link to the Github tests since we already used it for Windows. We just have to remove the Travis badge.

**Comment by ChrisRackauckas** [comment]
I mean below it says `CI / Julia 1 - ubuntu-latest - x86 (pull_request) ` instead of mentioning the environment.

**Comment by devmotion** [comment]
We used a standard name for the tests before (I think the same as the PkgTemplates defaults) and did not split tests into groups. I just removed the name.

**Pull request merged**